config/routes.rb

```diff
@@ -137,6 +137,9 @@
         scope module: :achievement do
           concerns :conditional
         end
+
+        get '/award' => 'achievements#award_users', on: :member
```

Comment:
/users makes sense, because we're saying we want the /course/id/achievements/id/users, and we want to replace them.

I'd call the controller achievement#awarded_users and achievement#award_users respectively, actually.